composer.json

```
@@ -66,7 +66,22 @@
         "refresh": [
             "rm -rf composer.lock vendor html/core html/modules/contrib html/profiles/contrib html/themes/contrib",
             "@composer update -W --ansi"
-        ]
+        ],
+        "docker-up": [
+            "docker compose -f docker-compose.nginx.yml -p nginx up -d",
+            "docker compose up -d --remove-orphans"
+        ],
+        "docker-shell": "test -f \".env\" || exit 1; export $(egrep -v '^#' .env | xargs); test -f \".env\"; sh -c \"docker exec -ti ${PROJECT_NAME}_web bash\"",
```

I accidentally left a test too many in here.

```suggestion
        "docker-shell": "test -f \".env\" || exit 1; export $(egrep -v '^#' .env | xargs); sh -c \"docker exec -ti ${PROJECT_NAME}_web bash\"",
```